config/src/test/java/org/springframework/security/config/annotation/web/configurers/AuthorizeHttpRequestsConfigurerTests.java

```diff
@@ -72,13 +72,28 @@ public void configureWhenAuthorizedHttpRequestsAndNoRequestsThenException() {
 						"At least one mapping is required (for example, authorizeHttpRequests().anyRequest().authenticated())");
 	}
 
+	@Test
```

Comment:
Thanks for adding these tests. Will you please add the equivalent of `configureMvcMatcherAccessAuthorizationManagerWhenNotNullThenVerifyUse`, just using the method you are introducing instead?